src/workerd/api/global-scope.c++

```diff
@@ -496,6 +496,9 @@ v8::Local<v8::String> ServiceWorkerGlobalScope::atob(kj::String data, v8::Isolat
 void ServiceWorkerGlobalScope::queueMicrotask(
     jsg::Lock& js,
     v8::Local<v8::Function> task) {
+  // TODO(later): It currently does not appear as if v8 attaches the continuation embedder data
```

Comment:
What is the TODO here?

It's not obvious to me that V8's behavior is wrong, and it seems easy enough to work around, so I'm not sure what we'd want to change here.

Reply:
It is wrong. The bug in v8 has been acknowledged by them. Essentially, v8 is supposed to be attaching the contextual information to all job types, including microtasks. This is relevant, for instance, in appropriately attaching async context to thenables (which use `EnqueueMicrotask`). The TODO here is for v8 to fix the issue, after which we can remove the workaround in this method.

In the meantime, I think it is safe for us to go ahead even tho thenables and microtasks aren't fully covered since it is possible to workaround in both cases.